section2/reflection.md

```diff
@@ -1,15 +1,33 @@
 
 2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
+### I currently demonstrate a Growth Mindset in the following ways:
+1. I believe I can improve my coding skills with effort and practice.
+2. I am pretty good at accepting feedback from others.
+
+### I currently don't demonstrate a Growth Mindset in the following ways:
+1. I need to be better at seeing my mistakes as a part of learning.
 
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
+* It was shocking to see, but I completely understand why it was done. It was uncomfortable because I had to stop what i was working on and go back to the old files and actually learn how what I was copying and pasting worked. Im happy that I now know what to type out to run a file in a node instead of relying on copying and pasting the correct way.
 
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
+1. A conditional statement is a line of code that instructs the computer on what decision to make when given a set of conditions.
+2. If there are dishes in the sink, I need to wash them. If there aren't dishes in the sink, I don't have to wash the dishes.
+3. If the password and username are correct, then you are successfully logged in. If the password or username aren't correct, you will be prompted to try again.
 
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
+* You add multiple conditions by using a logical operator such as "&&"
+* "&&" is read, by the computer as "and." so the computer would see the first condition and know that that condition needs to be met as well as the following condition because it read the "&&".
+* The computer would read this line of code `if ((cupsOfFlour <= 1) && (hasSauce = false)) {
+  console.log("I cannot make pizza");`as: if the variable cupsOfFlour is less than or equal to 1 AND the variable hasSauce is equal to FALSE, then console log the string "I cannot make pizza"
 
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
+* You have logical operators to check if two conditions are met by using "&&" and you have logical operators to check if one of two conditions are met by using "||"
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+7. What questions do you still have about `if` statements and/or functions?
+* Can you use "&&" and "||" in the same condition?
```

Comment:
Yep! You'll learn a lot more in Mod 1, but you can combine them. It can get a bit confusing and lead to unintended results based on order. This is where testing come in as a huge helper!